circle.yml

```diff
@@ -0,0 +1,24 @@
+machine:
+  python:
+    version: 2.7.12
```

Comment:
Yes, the build is [really slow](https://circleci.com/gh/openfisca/openfisca-core/8) otherwise.